src/main/java/org/candlepin/subscriptions/db/model/Host.java

```diff
@@ -304,6 +307,20 @@ public void setCloudProvider(String cloudProvider) {
         this.cloudProvider = cloudProvider;
     }
 
+    @PrePersist
+    void prePersist() {
+        if (Objects.isNull(displayName)) {
+            displayName = "";
```

Comment:
Theoretically, displayName shouldn't ever be null in the wild.  If somehow it is before a Host entity gets saved, default it to an empty string instead of `null`.  This is mostly to keep from having an overly-complex @Query in `HostRepository` with null & empty string checks for a scenario that doesn't happen.

Eventually we should probably enforce some constraints on the hosts.display_name column to solidify this.

Anyone think it's necessary to also attach a database update for `update hosts set display_name = '' where display_name is null`?  I wouldn't actually expect it to update any records, but figured I'd ask to just cross my T's and dot my I's.

Reply:
Instead of this, I was thinking we'd just do the equivalent in liquibase:

```xml
<addNotNullConstraint tableName="hosts" columnName="display_name" defaultNullValue="" />
```

Thoughts?